source/android/adaptivecards/.cxx/abi_configuration_2b4z1p0r.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I think we're not supposed to check in the `.cxx` folder (see github/gitignore#3155)